microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-1. Click the **Use Large Request Rows** checkbox.  The height of the rows in the table of network requests increases.
+1. In Visual Studio Code, open the `/travel-site/assets/img/optimized/` folder in the **Explorer**, which contains a copy of the four images displayed on the webpage, but already optimized for you. These images are 1000 pixels wide, and use the more optimized `.webp` format.
 
-   ![Large rows in the network requests table](../media/speed-glitch-tony-remix-network-use-large-request-rows.msft.png)
+    Resizing images depends on your operating system. For example, to resize images on Windows, you can use **PowerToys**. To learn more, see [Image Resizer utility](/windows/powertoys/image-resizer).
 
-1. If the **Size** column in the table of network requests isn't displayed, click the table header > **Size**.
+1. Open the `/travel-site/index.html` file and replace the four image paths as follows:
 
-Each **Size** cell shows two values.  The top value is the size of the downloaded resource.  The bottom value is the size of the uncompressed resource.  If the two values are the same, then the resource isn't being compressed when it is sent over the network.  For example, in the previous figure, the top and bottom values for `bundle.js` are `1.2 MB` and `1.2 MB`.
+    * Replace the four instances of `assets/img/` in the file with `assets/img/optimized`.
 
-Check for compression by inspecting the HTTP headers of a resource:
+    * Replace the four instances of `.jpg` in the file with `.webp`.
 
-1. Click `bundle.js`.
+1. In **Lighthouse**, click **Run an audit** (![Run an audit icon](./images/perform-icon.msft.png)) to go back to the main **Lighthouse** page without losing your baseline report.
 
-1. Click the **Headers** panel.
+1. Click **Analyze page load** again to see how the change affects load performance:
 
-   ![The Headers panel](../media/speed-glitch-tony-remix-network-use-large-request-rows-bundle-js.msft.png)
+    ![An Audits report after resizing images](./images/report-after-image-resize.png)
 
-1. Search the **Response Headers** section for a `content-encoding` header.  A `content-encoding` heading isn't displayed, meaning that `bundle.js` wasn't compressed.  When a resource is compressed, this header is usually set to `gzip`, `deflate`, or `br`.  For an explanation of the values, see [Directives](https://developer.mozilla.org/docs/Web/HTTP/Headers/Content-Encoding#Directives).
+Your score increased from 18 to 26. To verify how much data you saved, use the **Network** tool like you did before:
 
-Enough with the explanations; it's time to make some changes.  Enable text compression by adding a couple of lines of code, as follows.
+![The Network tool, showing the four image requests again, and the lower total amount of data transferred](./images/network-tool-images-after.png)
 
-1. In the editor tab, select **server.js**.
+Now, the images on the webpage only require to transfer 360 KB of data.
```

Comment:
```suggestion
Now, the images on the webpage only require transferring 360 KB of data.
```
instead of how much before?  I'd add like:

Now, the images on the webpage only require transferring 360 KB of data, instead of 1.234 MB.